benchmarks/backend/config/get_statistics.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env nu
```

Comment:
For this one it technically isn't required, however I would like to keep it consistent.

Reply:
I would prefer shell/bash code, I didn't see a package for nu in Debian :cry:

Reply:
You can see it here: https://www.nushell.sh/book/installation.html#installing-rust (I linked it in doc as well)
It is just way easier for me this way as I can easily execute groupings and parsing. It might be possible with bash as well, but that would take to long for me to figure out.

Reply:
Ahh, ok `cargo install nu` works, ok yes, then it is fine.